### PR TITLE
Update install-vamt.md

### PR DESCRIPTION
**Pull request by Karl-WE**
adding link to ADK, removing specific version to ease maintenance of this page as we would have to update it at least once a year.

### CODE REVIEW

File: windows/deployment/volume-activation/install-vamt.md [hunk 1]
```diff
@@ -49,8 +49,8 @@ You install VAMT as part of the Windows Assessment and Deployment Kit (ADK) for
 
 ### Install VAMT using the ADK
 
-1. Download and open the [Windows 10, version 1903 ADK](https://go.microsoft.com/fwlink/?linkid=2086042) package.
-Reminder: There won't be new ADK release for 1909.
+1. Download the latest version of [Windows 10 ADK](https://docs.microsoft.com/en-us/windows-hardware/get-started/adk-install).
+It is recommended to uninstall ADK and install the latest version, if you use a previous version. Existing data of VAMT is maintained in the respective VAMT database.
```
Comment (illfated) [inline windows/deployment/volume-activation/install-vamt.md:53]: The phrasing appears to be slightly off. I believe a reordering of the sentence clauses could help improve the clarity for English readers.
```suggestion
If an older version is already installed, it is recommended to uninstall the older ADK and install the latest version. Existing VAMT data is maintained in the VAMT database.
```
(I am fairly sure that "respective" is only useful when comparing multiple entities or versions.)

**Comment by illfated** [comment]
Yes, good idea to get a final review from the document author.

**Comment by Karl-WE** [comment]
Hello @Dansimp this has been merged but the change is not visible for unclear reasons. 
https://docs.microsoft.com/en-us/windows/deployment/volume-activation/install-vamt
cc @illfated any idea?

**Comment by illfated** [comment]
@Karl-WE : Looks correct & updated to me, maybe you are looking in the wrong paragraph? Here is a link to the commit, using the Rich Diff view:
- https://github.com/MicrosoftDocs/windows-itpro-docs/commit/c8c7f683e6977a23956012a2fa8861cef7c0c2e8?short_path=510612e#diff-510612e6dbbd7d3c801f53cc5b4347d92933eba943db95605299be392769a879

Compare that with the same paragraph in the current page:
- https://docs.microsoft.com/en-us/windows/deployment/volume-activation/install-vamt#install-vamt-using-the-adk

(Please note the direct paragraph link anchor, #install-vamt-using-the-adk, in the docs.microsoft.com link.)

Also note: if you still don't see the update, I am assume you may have a browser cache update issue, so you may want to refresh your browser cache.

**Comment by Karl-WE** [comment]
Hi @illfated when using your link I get this result.
image

**Comment by Karl-WE** [comment]
ok it does match but I missed to edit the requirements (d'oh)

**Pull request merged**